### PR TITLE
Fix filling out `time` questions

### PR DESCRIPTION
**Pull request by jkuester**
Fixes https://github.com/medic/cht-conf-test-harness/issues/249

I think this was probably a bug introduced in `3.0` (with the new Enekto changes) since I cannot re-create the behavior in `2.4.3`.  My guess is that Enekto added some extra logic to its time widget around localization (and apparently changed the `type` value).  I did a side-by-side comparison between `2.4.3` and `master` and confirmed that a form question that had the type `time` in `2.4.3` was now coming back with the type `text` in `master`.  

So, I have gone ahead and removed our special handling for the `time` type in `form-filler.js` and instead updated the `text` type handling to support filling out the time widget.  This code is closely related to the _datetime_ widget.  As a helpful reference, here is the form HTML you get for a _datetime_ widget:

```html
<div class="datetimepicker widget">
  <div class="date">
    <input class="ignore" type="text" placeholder="yyyy-mm-dd">
  </div>
  <div class="timepicker">
    <input class="ignore timepicker-default" type="text" placeholder="hh:mm">
    <button type="button" class="btn-icon-only btn-reset" aria-label="reset">
      <i class="icon icon-refresh"> </i>
    </button>
  </div>
</div>
```

And here is the HTML you get for just a _time_ widget:

```html
<div class="widget timepicker">
  <input class="ignore timepicker-default" type="text" placeholder="hh:mm">
  <button type="button" class="btn-icon-only btn-reset" aria-label="reset">
    <i class="icon icon-refresh"> </i>
  </button>
</div>
```

### CODE REVIEW

File: docs/Harness.html [hunk 1]
```diff
@@ -1054,7 +1054,7 @@ <h4 class="name" id="coreVersion"><span class="type-signature"></span>coreVersio
     
     <dt class="tag-source">Source:</dt>
     <dd class="tag-source"><ul class="dummy"><li>
-        <a href="harness.js.html">harness.js</a>, <a href="harness.js.html#line548">line 548</a>
+        <a href="harness.js.html">harness.js</a>, <a href="harness.js.html#line549">line 549</a>
```
Comment (kennsippell) [inline docs/Harness.html:1057]: this timestamp-only changes are kinda annoying. sorry for this

**Review by kennsippell — APPROVED** [review]

**Comment by kennsippell** [comment]
Thanks for taking this on.  Reminder to `npm version patch` before merging

**Comment by jkuester** [comment]
@kennsippell what is the release process here?  I pushed the version change to the `package.json` as suggested, but I did not actually push the git tag (I assume we want the tag on `master` and not on `249_time_input`).  

Should I merge this PR and then someone (aka you) will handle actually tagging the release and publishing it on NPM?

**Comment by kennsippell** [comment]
@jkuester I've merged and published. Pretty ghetto. I literally just run `npm publish`. Pursuing improvements in https://github.com/medic/cht-conf-test-harness/issues/237 but no progress in a long while.

**Pull request merged**